src/redux/migrations.ts

```diff
@@ -1218,4 +1218,28 @@ export const migrations = {
       pushNotificationRequestedUnixTime: 1692878055000,
     },
   }),
+  145: (state: any) => ({
+    ...state,
+    transactions: {
+      ...state.transactions,
+      standbyTransactions: (state.transactions.standbyTransactions as StandbyTransaction[]).map(
+        (tx) => {
+          return { ...tx, chain: Chain.Celo }
+        }
+      ),
+      transactions: (state.transactions.transactions as TokenTransaction[]).map((tx) => {
+        const __typename = // @ts-ignore
+          tx.__typename === 'TokenTransferV2'
+            ? 'TokenTransferV3' // @ts-ignore
+            : tx.__typename === 'NftTransferV2'
+            ? 'NftTransferV3'
+            : 'TokenExchangeV3'
```

Comment:
Is it safe to change these in place? Or should we just drop them and let the app fetch and build this list again?

Reply:
My assumption was that it was safe, since the rest of the app has been updated to handle these new typenames.